core/src/main/java/de/muenchen/allg/itd51/wollmux/former/ComboboxMergeDescriptor.java

```diff
@@ -28,22 +28,22 @@
 import de.muenchen.allg.itd51.wollmux.former.model.IdModel;
 
 /**
- * Enthält Informationen über eine erfolgte verschmelzung mehrerer Checkboxen zu
- * einer einzigen Combobox.
+ * Contains information about a happened fusing together of multiple checkboxes to
+ * a single combobox.
  */
 public class ComboboxMergeDescriptor
 {
 
   /**
-   * Das aus dem Merge neu hervorgegangene {@link FormControlModel}.
+   * The arised {@link FormControlModel} from the merger.
    */
   private FormControlModel combo;
 
   /**
-   * Eine {@link Map}, deren Schlüssel die {@link IdModel}s der Checkboxen
-   * sind, die verschmolzen wurden, wobei jede dieser IDs auf einen String gemappt
-   * wird, der den ComboBox-Wert beschreibt, den auszuwählen dem Aktivieren der alten
-   * Checkbox entspricht.
+   * A {@link Map}, which keys are the {@link IdModel}s of the checkboxes,
+   * which have been merged, in which every of these IDs were mapped on a string,
+   * which describes the combobox value. The choose one of them matches the activation of the old
```

Comment:
That's completely mangled. "Selecting one of those strings corresponds to activating the respective, previous checkbox"